### ISSUE TITLE
fix Talladega

### ISSUE THREAD

**Issue by ExodusTheHuman** [memeee310/FNF-VS-Shaggy-Web#2]
Hey could you like fix Talladega in freeplay, whenever it loads it just shows the stage and characters but nothing happens and the screen freezes. I've tried using the lowest graphics quality as I could for better performance but it still doesn't work. It might be a coding issue idk :/

**Comment by JustSpaceisded** [comment]
> Hey could you like fix Talladega in freeplay, whenever it loads it just shows the stage and characters but nothing happens and the screen freezes. I've tried using the lowest graphics quality as I could for better performance but it still doesn't work. It might be a coding issue idk :/

yeah i agree its a 9 key song and ive beat all songs with FullCombo and i want to do the hardest song and it frezzes pls fix yo.

**Comment by memeee310** [comment]
I am sorry but at the moment, I have no earthly clue how to fix it. I am surprised it even loaded up.

**Comment by JustSpaceisded** [comment]
ok

**Comment by JustSpaceisded** [comment]
if its an html or javascript problem i could fix im not that good at the other things tho.
